infra/src/main/scala/scaladex/infra/storage/sql/tables/ProjectDependenciesTable.scala

```diff
@@ -29,4 +29,10 @@ object ProjectDependenciesTable {
       limit = Some(limit)
     )
   }
+  val deleteSourceProject: Update[Project.Reference] =
```

Comment:
It reads like we are deleting the source project rather than the dependencies of the source project.

```suggestion
  val deleteBySource: Update[Project.Reference] =
```